test/index.js

```diff
@@ -8,9 +8,11 @@ describe('Lob', function () {
   });
 
   it('allows the use of promises and callbacks', function (done) {
-    Lob.countries.list()
-    .then(function (body) {
-      expect(body.data).to.be.instanceof(Array);
+    var Lob = require('../lib')(API_KEY);
+
+    Lob.addresses.list()
+    .then(function(result) {
```

Comment:
nit: arrow function

Reply:
I actually want to keep the consistency of using the function keyword in this file... so how about I change the other part of the diff back to using `function`?

Reply:
yeah it's up to you, we're going to es6ify this repo soon anyways!